src/components/FacetInsights.js

```diff
@@ -167,12 +185,18 @@ export default class FacetInsights extends React.Component<FacetInsightsDefaultP
     ObjectUtils.removeItem(facetOrder, 'keyphrases');
     ObjectUtils.removeItem(facetOrder, 'date');
 
+    const facetFilters = searcher.state.facetFilters;
```

Comment:
Prefer [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment).
```suggestion
    const { facetFilters } = searcher.state;
```